data/World/Spirit Temple MQ.json

```diff
@@ -31,8 +37,14 @@
             "Spirit Temple MQ Silver Block Hallway Chest": "
                 has_bombchus and (Small_Key_Spirit_Temple, 7) and Slingshot and
                 (can_use(Dins_Fire) or
-                at('Adult Spirit Temple', (can_use(Fire_Arrows) or
-                (logic_spirit_mq_frozen_eye and can_use(Bow) and can_play(Song_of_Time)))))",
+                    at('Adult Spirit Temple', (can_use(Fire_Arrows) or
+                        (logic_spirit_mq_frozen_eye and can_use(Bow) and can_play(Song_of_Time)))))",
+            "Spirit Temple MQ Child Torch Slugs Room Pot": "True",
+            "Spirit Temple MQ Child 3 Gibdo Room Pot 1": "(Sticks or Kokiri_Sword) and has_bombchus and Slingshot",
+            "Spirit Temple MQ Child 3 Gibdo Room Pot 2": "(Sticks or Kokiri_Sword) and has_bombchus and Slingshot",
+            "Spirit Temple MQ Child Stalfos Fight Pot 1": "(Sticks or Kokiri_Sword) and has_bombchus and Slingshot",
+            "Spirit Temple MQ Child Stalfos Fight Pot 2": "(Sticks or Kokiri_Sword) and has_bombchus and Slingshot",
+            "Spirit Temple MQ Child Stalfos Fight Pot 3": "(Sticks or Kokiri_Sword) and has_bombchus and Slingshot",
```

Comment:
Requested Change: So to get to this room you only need to kill the Slugs which we consider in logic with sticks, sword, or bombs. Requiring sticks/sword for the enemies chest was just to help you a little since you do have to fight so many enemies to get there. Maybe it should be only sword b/c one of the enemies is a Stalfos, but that Stalfos in particular has a lot of ways to defeat it, so I guess it's fine, but I digress. The new problem actually isn't the slugs, it's hitting all of these switches with the Gibdo -right there-, lol. Din's Fire also works on Gibdos, so I suggest that as another alternative to sticks/sword. It's four casts but these guys always drop some magic.